### DISCUSSION TITLE
Welcome to AI-Radio Discussions!

### DISCUSSION THREAD

**Discussion by iampawan** [iampawan/AI-Radio#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robinrishi21** [comment]
how to deal with null safety feature !!

**Comment by Vikash620404** [comment]
There is a issue going on with your project, please help us. We are facing an error says lateInitializationError, it is happening so beacuse radios is unable to fetch other files data. Even after writing all this:- class HomePageState extends State<HomePage> {
  List<MyRadio> radios;

  void initState() {
    super.initState();
    fetchRadios();
  }

  fetchRadios() async {
    final radioJson = await rootBundle.loadString("assets/radio.json");
    radios = MyRadioList.fromJson(radioJson).radios;
    



please if you are reading this help us out🙏🙏🙏🙏
